pandas/core/frame.py

```diff
@@ -3835,34 +3835,59 @@ def sortlevel(self, level=0, axis=0, ascending=True, inplace=False,
                                inplace=inplace, sort_remaining=sort_remaining)
 
     def nlargest(self, n, columns, keep='first'):
-        """Get the rows of a DataFrame sorted by the `n` largest
-        values of `columns`.
+        """
+        Return the `n` largest rows ordered by `columns`.
+
+        Return the `n` largest rows of `columns` in descending order. The
+        remaining columns, although not used for ordering, are returned as
+        well.
 
         Parameters
         ----------
         n : int
-            Number of items to retrieve
+            Number of items to retrieve.
         columns : list or str
-            Column name or names to order by
+            Column name or names to retrieve values from.
         keep : {'first', 'last'}, default 'first'
```

Comment:
Default value is implied for first element in the possible values, so no need to explicitly say `default 'first'` again

Reply:
@WillAyd I think we changed the guide on this (some time last week) .. :) 
For explicitness, I think it is good to keep the " default 'first' ". A new reader cannot know the rule of "default is fist value in the set of options"